File: python/Bucle while.py
# en un ciclo while  crear un contador y en cada vuelta imprime un mensaje

cont = 1

while cont >= 10:
    print ("Hola MUndo  " , cont)
    cont  += 1 
    
#otro ejercicio para adivinar un numero

num = int(input("Digite un numero. "))

while num != 8:
    
    if num < 8:
        print("Numero menor digite nuevamente el numero.")
        num = int(input("Digite un numero. "))
    elif num >8:
        print("Numero mayor digite nuevamente el numero.")
        num = int(input("Digite un numero. "))
8
print("Adivinaste el numero.  ")        

File: javaScript/bucle while.js
let cont = 1;

while (cont <= 10) {
    console.log("Hola Mundo " + cont);
    cont += 1; // Incrementar el contador
}

//otro ejercicio

let num = parseInt(prompt("Digite un número."));

while (num !== 8) {
    if (num < 8) {
        console.log("Número menor, digite nuevamente el número.");
    } else if (num > 8) {
        console.log("Número mayor, digite nuevamente el número.");
    }
    num = parseInt(prompt("Digite un número."));
}

console.log("Adivinaste el número.");